src/components/create_cluster/index.js

```diff
@@ -178,6 +169,14 @@ class CreateCluster extends React.Component {
     this.cluster_name.select();
   }
 
+  isScalingAutomatic(provider, releaseVer) {
+    if (provider != 'aws') {
+      return false;
+    }
+
+    return (cmp(releaseVer, '6.1.0') === 1);
```

Comment:
Since the `cmp` function is so un-intuitive, I'd like a comment line above to explain what this means (releaseVer >= 6.1.0 I assume)

Reply:
Added.